platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/offline/OfflineActivity.java

```diff
@@ -264,6 +264,7 @@ public void onStatusChanged(OfflineRegionStatus status) {
         if (status.isComplete()) {
           // Download complete
           endProgress("Region downloaded successfully.");
+          offlineRegion.setDownloadState(OfflineRegion.STATE_INACTIVE);
```

Comment:
@tobrun Ref counting aside, this looks like new behavior for region downloads. Are there any side effects if a developer (like until now) doesn't manually set the region as inactive?

Reply:
This will result in not pausing the Filesource, this will result int the same behavior as it is today. For this reason I'm not seeing this as a semver change but an enhancement.

Reply:
@tobrun Makes sense, thanks for the clarification.